src/commands/compress.rs

```diff
@@ -83,6 +83,7 @@ pub fn compress_files(
                 zstd_encoder.multithread(num_cpus::get_physical() as u32)?;
                 Box::new(zstd_encoder.auto_finish())
             }
+            Brotli => Box::new(brotli::CompressorWriter::new(encoder, BUFFER_CAPACITY, 11, 22)),
```

Comment:
Have to admit, the last two params are kinda random. I picked them because they were used in [this example](https://docs.rs/brotli/7.0.0/src/compress/compress.rs.html#12) but I have no idea what they do. They are called `q: u32, lgwin: u32`.